src/less/carousel/base/carousel.less

```diff
@@ -68,6 +68,20 @@
     }
 }
 
+.carousel__container--image-treatment {
+    .carousel__list img {
+        display: inline-block;
+        object-fit: contain;
+        max-width: 100%;
```

Comment:
Do we need `max-height` as well?

Reply:
I don't think so, with all of the test cases I tried, the pictures stay within the height bounds because of flexbox. Unless we want to change how we are doing that